feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/main/StakingViewState.kt

```diff
@@ -332,3 +363,16 @@ class WelcomeViewState(
         return rewardCalculator.await()
     }
 }
+
+fun Long.formatTime(): String {
+    val totalSeconds = this / 1000
+    val hours = totalSeconds / 3600
+    val minutes = (totalSeconds % 3600) / 60
+    val seconds = totalSeconds % 60
+    return "%02d:%02d:%02d".format(hours, minutes, seconds)
```

Comment:
There is already method in resource manager

Reply:
I could not managed to get ResourceManager from view without di fast. For now I propose to leave it as it is and to create refactoring task for this format